### DISCUSSION TITLE
pvp stuff

### DISCUSSION THREAD

**Discussion by BeeRando** [Zrips/CMI#2360] [Feature Requests]
Some commands could be
 - /cmi pvptoggle
 - /cmi pvpon
 - /cmi pvpoff

Perms could be
 - pvpbypass (lets you attack people who have their pvp toggled off)
 - pvpoff (anyone with this is safe from pvp, not having to type teh command)

**Comment by lwd-adam** [comment]
This is already requested in https://github.com/Zrips/CMI/issues/2249
